### ISSUE TITLE
Unused queen8 example/problem in functions chapter directory

### ISSUE THREAD

**Issue by StevenMaude** [miekg/learninggo#23]
Maybe [this code](https://github.com/miekg/learninggo/blob/361e6e5e151ca25257852cec1a2333b55b65feb5/ex/functions/src/queen8.go) is a problem solution that never had a problem written for it?

**Comment by miekg** [comment]
Yes, could very well be. Or I pulled it out. Feel free to make it into a problem and send a PR.
